lsql/judge/feedback.py

```diff
@@ -50,10 +50,47 @@ def feedback_headers(expected, obtained):
     """
     if expected['header'] == obtained['header']:
         return ''
-    return render_to_string('feedback_wa_headers.html',
-                            {'expected': header_to_str(expected['header']),
-                             'obtained': header_to_str(obtained['header'])}
-                            )
+
+    if len(expected['header']) != len(obtained['header']) and expected['header'] != obtained['header']:
```

Comment:
A la fuerza

    and expected['header'] != obtained['header']

es verdad porque porque si `expected['header'] == obtained['header']` habrías salido en la línea 51

Reply:
¿Has considerado este comentario? Si estoy equivocado avísame

Reply:
Lo cambie abajo, no ahí ahora voy